mmdet/datasets/pipelines/transforms.py

```diff
@@ -1903,3 +1905,195 @@ def __repr__(self):
                      else f'cutout_shape={self.candidates}, ')
         repr_str += f'fill_in={self.fill_in})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class RandomAffine:
+    """Random affine transform data augmentation.
+
+    This operation randomly generates affine transform matrix which including
+    rotation, translation, shear and scaling transforms.
+
+    Args:
+        rotate_degree (float): Maximum degrees of rotation transform.
+            Default: 10.
+        translate_ratio (float): Maximum ratio of translation. Default: 0.1.
+        scaling_ratio (tuple[float]): Min and max ratio of scaling transform.
```

Comment:
`scaling_ratio` sounds like it is a float number
Can we rename it to scaling_ratio_range